### PR TITLE
BO-126 Deploy with firebase

### PR DESCRIPTION
**Pull request by njam**

### CODE REVIEW

**Comment by njam** [comment]
The website is deployed to firebase already: https://cargomedia-website.firebaseapp.com/

Once it's ready I will switch the production domain.

File: README.md [hunk 1]
```diff
@@ -1,66 +1,53 @@
-cargomedia.github.io
```
Comment (christopheschwyzer) [inline README.md]: So we rename the repo as well?

**Comment by christopheschwyzer** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (njam) [inline README.md]: I would suggest yes. I will rename it once I get the "ok" on the review. Please click the "thumbs up" button :D

File: README.md [hunk 2]
```diff
@@ -1,66 +1,53 @@
...
-====================
+cargomedia-website
+==================
+Company website - http://www.cargomedia.ch/
```
Comment (ppp0) [inline README.md]: May be we need to adapt this URL too and replace `cargomedia.github.io` with `cargomedia-website` ?

**Pull request merged**